poetry/patterns/single.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * Title: single
+ * Slug: poetry/single
+ * Categories: hidden
```

Comment:
we don't add categories if the pattern is hidden (it doesn't exist as a default category and if we wanted to create one, we would have to register it first)